config/action.d/iptables-allports.conf

```diff
@@ -21,6 +23,10 @@ actionstart = <iptables> -N f2b-<name>
               <iptables> -A f2b-<name> -j <returntype>
               <iptables> -I <chain> -p <protocol> -j f2b-<name>
 
+actionstart6 = <ip6tables> -N f2b-<name>6
+               <ip6tables> -A f2b-<name>6 -j <returntype>
+               <ip6tables> -I <chain> -p <protocol> -j f2b-<name>6
```

Comment:
it is just that without custom suffix you can note that both of the invocations are identical, and only command is different. so we are arriving to some elderly discussion on may be possibly somehow allowing for a single specification ... see e.g. https://github.com/fail2ban/fail2ban/commit/28b44d2655abdf3768b30751a581def2f5f4b5a5 (referenced from https://github.com/fail2ban/fail2ban/issues/39#issuecomment-4324711) .  Just wonder if you have thoughts on that?